### ISSUE TITLE
 in windows and laravel 8 + php8.1 error

### ISSUE THREAD

**Issue by f4soft** [sindresorhus/capture-website-cli#41]
php artisan queue:work
[2022-12-21 17:59:56][72] Processing: App\Jobs\LinkScreenshot
node:events:491
      throw er; // Unhandled 'error' event
      ^

Error: EOF: end of file, write
    at Socket._write (node:internal/net:61:25)
    at writeOrBuffer (node:internal/streams/writable:392:12)
    at _write (node:internal/streams/writable:333:10)
    at Writable.write (node:internal/streams/writable:337:10)
    at file:///C:/Users/f4soft/AppData/Roaming/npm/node_modules/capture-website-cli/cli.js:289:18
Emitted 'error' event on Socket instance at:
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -4095,
  syscall: 'write',
  code: 'EOF'
}

Node.js v18.12.1

**Comment by f4soft** [comment]
calling from php:

        $result = shell_exec(
            sprintf(
                "capture-website %s --scale-factor=1 --type=webp --width 1000",
                escapeshellcmd($urlToCapture)
            )
        );

**Comment by f4soft** [comment]
with #42 work all ok

**Comment by sindresorhus** [comment]
I don't think this is a problem with this package, if it is, submit a failing test.
